### PR TITLE
uptimed:add plist_options and update test

### PR DESCRIPTION
**Pull request by GauthamGoli**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/uptimed.rb [hunk 1]
```diff
@@ -54,6 +56,8 @@ def plist; <<-EOS.undent
   end
 
   test do
-    system "#{bin}/uprecords"
+    system "#{opt_sbin}/uptimed", "-t", "0"
```
Comment (MikeMcQuaid) [inline Formula/uptimed.rb]: Use just `sbin` here.

File: Formula/uptimed.rb [hunk 2]
```diff
@@ -54,6 +56,8 @@ def plist; <<-EOS.undent
...
+    sleep(2.0)
```
Comment (MikeMcQuaid) [inline Formula/uptimed.rb]: `sleep 2`

File: Formula/uptimed.rb [hunk 3]
```diff
@@ -54,6 +56,8 @@ def plist; <<-EOS.undent
...
+    assert(shell_output("#{bin}/uprecords -s | awk '/->/ {print $3}'").to_i>=0, "Uptime returned is invalid")
```
Comment (MikeMcQuaid) [inline Formula/uptimed.rb]: Use `assert_match` instead of `awk` here.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Thanks for the PR! A few comments here.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again @GauthamGoli!

**Comment by GauthamGoli** [comment]
Thanks for the code review!

**Pull request merged**